src/components/EmailNotifications/EmailNotificationsForm.js

```diff
@@ -0,0 +1,302 @@
+                  `}
+                >
+                  <Button
+                    mode="strong"
+                    disabled={emailInvalid || !termsAccepted}
+                    onClick={handleOnSubscribeToNotifications}
+                    size="medium"
+                  >
+                    {existingEmail ? 'Update' : 'Subscribe'}
+                  </Button>
+                </div>
+              )}
+            </div>
+
+            {emailInvalid && (
+              <div>
+                <p
+                  css={`
+                    color: ${theme.negative};
+                    ${textStyle('body4')};
+                    text-align: left;
+                    height: 0;
+                    margin-top: ${0.5 * GU}px;
+                  `}
+                >
+                  Please enter a valid email address.
+                </p>
+              </div>
+            )}
+          </div>
+        </div>
+        <LegalTermsAndPolicy
+          termsAccepted={termsAccepted}
+          setTermsAccepted={setTermsAccepted}
+        />
+
+        {isModal && (
+          <div
+            css={`
+              display: flex;
+              justify-content: space-between;
+              flex-direction: ${compactMode ? 'column' : 'row'};
+              width: 100%;
+              margin-bottom: ${1.5 * GU}px;
+              margin-top: ${3 * GU}px;
+            `}
+          >
+            <ActionButtons compactMode={compactMode} onClick={handleOnOptOut}>
+              Opt out
+            </ActionButtons>
+            <ActionButtons
+              compactMode={compactMode}
+              mode="strong"
+              disabled={emailInvalid || !termsAccepted}
+              onClick={handleOnSubscribeToNotifications}
+            >
+              Subscribe
+            </ActionButtons>
+          </div>
+        )}
+      </div>
+    </>
+  )
+}
+
+function TextContent({ update, account }) {
+  const theme = useTheme()
+  const content = update
+    ? `Enter a new email address for your account ${account}. We will continue sending email notifications to the current email address until you verify this new email address.`
+    : `Associate an email address to your account ${account}, so you can get notifications from all Aragon Court events.`
+
+  return (
+    <span
+      css={`
+        ${textStyle('body2')};
+        color: ${theme.surfaceContentSecondary};
+        margin-top: ${1.5 * GU}px;
+      `}
+    >
+      {transformAddresses(content, (part, isAddress, index) =>
+        isAddress ? (
+          <span title={part} key={index}>
+            <LocalIdentityBadge
+              connectedAccount={addressesEqual(part, account)}
+              entity={part}
+              compact
+            />
+          </span>
+        ) : (
+          <span key={index}>{part}</span>
+        )
+      )}
+    </span>
+  )
+}
+
+function LegalTermsAndPolicy({ termsAccepted, setTermsAccepted }) {
```

Comment:
I noticed that but didn't want to create a component for that since is a really simple one, where in one of the cases we are just displaying a text with links inside